### DISCUSSION TITLE
Serendipity Styx 4.0.0 release

### DISCUSSION THREAD

**Discussion by ophian** [ophian/styx#37] [General]
### Serendipity Styx 4.0.0 Release

Please read the important changeLog **NEWS** (1) file for changes since Serendipity Styx 3.9.1 or the real commit **history** (2).

[1] https://github.com/ophian/styx/blob/4.0.0/docs/NEWS
[2] https://github.com/ophian/styx/commits/4.0.0

It'll be [announced](https://ophian.github.io/blog) on the Styx [Webpage](https://ophian.github.io/)

**Full Changelog**: https://github.com/ophian/styx/compare/3.9.1...4.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/ophian/styx/releases/tag/4.0.0'>Serendipity Styx 4.0.0 release</a>.</em>

**Comment by beatsblog** [comment]
Thanks for your patient and passionate work! Great job!

**Comment by hbarel** [comment]
Congratulations, @ophian !
I shall give it a try shortly.
Your commitment is highly appreciated.

**Comment by ophian** [comment]
I just released a minor END-YEAR-RELEASE as 4.0.1. Happy to have you onboard!
Happy NEW Year and lots of blog posts in 2023 ! 😀
[Serendipity Styx 4.0.1 release](https://github.com/ophian/styx/releases/tag/4.0.1)

**Comment by hbarel** [comment]
I have installed 4.0.1 and have to report that so far I see no major issues!
